.prettierrc.js

```diff
@@ -4,4 +4,6 @@ module.exports = {
   trailingComma: 'es5',
   printWidth: 100,
   tabWidth: 2,
+  printWidth: 120,
```

Comment:
I think `printWidth` is already written above(line#5), please update it.

Reply:
Yeah, I noticed that and removed the additional one.